### PR TITLE
blueprints updated for engineers unlock

### PR DESCRIPTION
**Pull request by vetall812**
done for materials
what about commodities needed for some of engineers?

### CODE REVIEW

File: EDEngineer/Resources/Data/blueprints.json [hunk 1]
```diff
@@ -16406,5 +16406,65 @@
...
     "Grade": 5
+  },
+  {
+    "Type": "The Sarge",
+    "Name": "unlock",
+    "Engineers": [ "@Unlock" ],
+    "Ingredients": [
+    "Name": "Aberrant Shield Pattern Analysis",
+    "Size": 50
+    ],
```
Comment (msarilar) [inline EDEngineer/Resources/Data/blueprints.json]: this is not valid json, it should look like this :

```
    "Ingredients": [
      {
        "Name": "Modified Consumer Firmware",
        "Size": 1
      }],
```

File: EDEngineer/Resources/Data/blueprints.json [hunk 2]
```diff
@@ -16406,5 +16406,65 @@
       }
     ],
     "Grade": 5
+  },
+  {
+    "Type": "The Sarge",
+    "Name": "unlock",
```
Comment (msarilar) [inline EDEngineer/Resources/Data/blueprints.json]: could you swap these two? Type = Unlock and Name = The Sarge
it makes more sense that way

Comment (vetall812) [inline EDEngineer/Resources/Data/blueprints.json]: sure 1 sec

File: EDEngineer/Resources/Data/blueprints.json [hunk 3]
```diff
@@ -16406,5 +16406,65 @@
       }
     ],
     "Grade": 5
+  },
+  {
+    "Type": "The Sarge",
+    "Name": "unlock",
+    "Engineers": [ "@Unlock" ],
```
Comment (msarilar) [inline EDEngineer/Resources/Data/blueprints.json]: Engineer can be The Sarge himself :)

Comment (vetall812) [inline EDEngineer/Resources/Data/blueprints.json]: hmmm... i've thought about it. but it breaks logic with "synthesis"

Comment (msarilar) [inline EDEngineer/Resources/Data/blueprints.json]: synthesis basically means the player himself ; the main idea is to have a dedicated filter

with engineer = The Sarge, users will be able to see everything related to The Sarge by filtering blueprints with his name ; and in the Type filters, they will have the "Unlock" type as well

Comment (vetall812) [inline EDEngineer/Resources/Data/blueprints.json]: well... if you wish =) 
one sec more

Comment (vetall812) [inline EDEngineer/Resources/Data/blueprints.json]: done.

**Comment by msarilar** [comment]
Thanks a bunch :)

File: EDEngineer/Resources/Data/blueprints.json (hunk 3 above)
Comment (vetall812) [inline EDEngineer/Resources/Data/blueprints.json]: //it'll make my parsing a bit... strange =)

**Pull request merged**